lib/pharos/phases/join_node.rb

```diff
@@ -23,6 +23,7 @@ def call
         end
         join_command << "--node-name #{@host.hostname}"
         join_command << "--ignore-preflight-errors DirAvailable--etc-kubernetes-manifests"
+        join_command << "--ignore-preflight-errors SystemVerification"
```

Comment:
Ignores docker version validation.

Reply:
maybe add a comment why the system checks are ignored